### PR TITLE
Fix: Lightbox external links

### PR DESCRIPTION
**Pull request by crhallberg**
[VUFIND-1523](https://vufind.org/jira/browse/VUFIND-1523)

### CODE REVIEW

File: themes/bootstrap3/js/lightbox.js [hunk 1]
```diff
@@ -250,10 +250,14 @@ VuFind.register('lightbox', function Lightbox() {
    */
   _constrainLink = function constrainLink(event) {
     var $link = $(this);
+    var urlRoot = location.origin + "/" + VuFind.path;
+
     if (typeof $link.data("lightboxIgnore") != "undefined"
       || typeof $link.attr("href") === "undefined"
       || $link.attr("href").charAt(0) === "#"
       || $link.attr("href").match(/^[a-zA-Z]+:[^/]/) // ignore resource identifiers (mailto:, tel:, etc.)
+      || ($link.attr("href").slice(0, 4) == "http" // external links
```
Comment (demiankatz) [inline themes/bootstrap3/js/lightbox.js]: Is there a reason to use `==` instead of `===` here? GitHub Actions is complaining about it.

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @crhallberg -- one very small question.

File: themes/bootstrap3/js/lightbox.js (hunk 1 above)
Comment (crhallberg) [reply themes/bootstrap3/js/lightbox.js]: For string comparison there is no difference so == is usually clearer but that's not what our styles enforce - will fix.

Comment (crhallberg) [reply themes/bootstrap3/js/lightbox.js]: Done.

**Comment by demiankatz** [comment]
@crhallberg, I did some hands-on testing, and this wasn't quite working right. I think I've fixed it with 7109b7b -- there was an extra slash in the constructed base URL, and the `indexOf` method wasn't capitalized correctly. I think it's all good now, but do you mind double-checking my work?

**Review by demiankatz — APPROVED** [review]

**Comment by demiankatz** [comment]
Thanks for the double-check, @crhallberg -- merging now!

**Pull request merged**